### ISSUE TITLE
[administration] Unable to change Customer address

### ISSUE THREAD

**Issue by patpilus** [shopware/shopware#692]
**Description**  
Shopware version: 6.1.4

When trying to `Apply` changes in Customer's address...:
![image](https://user-images.githubusercontent.com/13252273/77536910-74fed500-6e9d-11ea-8ed9-b0dcedd9dd11.png)
 ... I'm getting such error in browser's console:
```
vendors-node.js?15851359021360952:1 TypeError: Cannot read property 'forEach' of undefined
    at Object.getRequiredProperties (commons.js?1585135902166226:1)
    at s.isValidAddress (app.js?15851359023888663:1)
    at s.onSaveAddress (app.js?15851359023888663:1)
    at Xe (vendors-node.js?15851359021360952:1)
    at HTMLButtonElement.n (vendors-node.js?15851359021360952:1)
    at HTMLButtonElement.o._wrapper (vendors-node.js?15851359021360952:1)
```

As a result modified address is not updated.

**Comment by pantrtxp** [comment]
Hi @patpilus 

This issue is fixed on the master already and will be shipped with the upcoming 6.2 release

Sunny regards
Pantrtxp

**Comment by pantrtxp** [comment]
@patpilus is it okay to close the issue? What do you think?

Sunny regards
Pantrtxp

**Comment by patpilus** [comment]
If it will be fixed in 6.2 release then it's fine for me :)
